.travis.yml

```diff
@@ -70,7 +71,7 @@ matrix:
         # Needs python prefix to use the correct interpretor.
         - python ./obs_clone.py --cache --debug --apiurl-target local
       script:
-        - nosetests --with-coverage --cover-package=. --cover-inclusive
+        - nosetests --with-coverage --cover-package=. --cover-inclusive --exclude-dir=./oqamaint
```

Comment:
What's the reason to not include the `./oqamaint` directory? Previously, only `osclib` was included and gave a rather false sense of coverage.

Reply:
links between packages.